Kotlin/app/build.gradle

```diff
@@ -50,7 +50,7 @@ android {
 
 dependencies {
     // Import Taboola SDK
-    implementation 'com.taboola:android-sdk:4.0.8'
+    implementation 'com.taboola:android-sdk-beta:4.0.17-rc5-945-aa3b54952'
```

Comment:
@filippetrovic-taboola don't forget to change it to an official version once released.